docs/deploying.md

```diff
@@ -12,145 +12,123 @@ weight=3
 
-That's it.
+You can now use it with docker:
 
-You can now tag an image and push it:
+	# Get any image from the hub and tag it to point to your registry:
+    docker pull ubuntu && docker tag ubuntu localhost:5000/batman/ubuntu
 
-    $ docker pull ubuntu && docker tag ubuntu localhost:5000/batman/ubuntu
-    $ docker push localhost:5000/batman/ubuntu
+	# ... then push it to your registry:
+    docker push localhost:5000/batman/ubuntu
 
-Then pull it back:
+	# ... then pull it back from your registry:
+    docker pull localhost:5000/batman/ubuntu
 
-    $ docker pull localhost:5000/batman/ubuntu
+Now, stop your registry:
 
-## Where is my data?
+    docker stop registry && docker rm registry
 
-By default, your registry stores its data on the local filesystem, inside the container.
+## Storage
 
-In a production environment, it's highly recommended to use [another storage backend](storagedrivers.md), by [configuring it](configuration.md).
+By default, your registry data is stored *inside the container* - thus it will be lost if you remove the container.
 
-If you want to stick with the local posix filesystem, you should store your data outside of the container.
+If you want your data to persist, and in order to achieve decent I/O performance, you should mount a host volume inside the container and instruct the registry to use that folder:
 
-This is achieved by mounting a volume into the container:
-
-     $ docker run -d -p 5000:5000 \
+	docker run -d -p 5000:5000 --restart=always --name registry \
         -e REGISTRY_STORAGE_FILESYSTEM_ROOTDIRECTORY=/var/lib/registry \
-        -v /myregistrydata:/var/lib/registry \
-        --restart=always --name registry registry:2
-
-## Making your Registry available
+        -v `pwd`/data:/var/lib/registry \
+        registry:2
 
-Now that your registry works on `localhost`, you probably want to make it available as well to other hosts.
+### Alternatives
 
-Let assume your registry is accessible via the domain name `myregistrydomain.com` (still on port `5000`).
+You might consider using [another storage backend](https://github.com/docker/distribution/blob/master/docs/storagedrivers.md) instead of the local filesystem, by [configuring it](https://github.com/docker/distribution/blob/master/docs/configuration.md#storage).
 
-If you try to `docker pull myregistrydomain.com:5000/batman/ubuntu`, you will see the following error message:
 
-```
-FATA[0000] Error response from daemon: v1 ping attempt failed with error:
-Get https://myregistrydomain.com:5000/v1/_ping: tls: oversized record received with length 20527. 
-If this private registry supports only HTTP or HTTPS with an unknown CA certificate,please add 
-`--insecure-registry myregistrydomain.com:5000` to the daemon's arguments.
-In the case of HTTPS, if you have access to the registry's CA certificate, no need for the flag;
-simply place the CA certificate at /etc/docker/certs.d/myregistrydomain.com:5000/ca.crt
-```
+## Running a domain registry
```

Comment:
So, I want to distinguish between "localhost" registries, and registries exposed to other hosts - both are "private", but one is more widely available.

Wdyt should be the proper wording?